config/prow/plugins.yaml

```diff
@@ -1081,3 +1081,21 @@ external_plugins:
     events:
       - issue_comment
       - pull_request
+  kubernetes-sigs/cluster-api:
```

Comment:
@saschagrunert @justaugustus I think we can remove this from release engineering concern right now given that currently none of these repos where the plugin would be enabled our under or purview. What do you think?

Reply:
Agree :+1:

Reply:
thanks @hasheddan @saschagrunert